db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Provisioning/StateMachines/ProvisionRequestApproval.class/__methods__/validate_request.rb

```diff
@@ -51,10 +81,7 @@
 
 # Validate max_cpus if not nil or empty
 unless max_cpus.blank?
-  desired_cpus = prov_resource.get_option(:number_of_cpus).to_i
-  if desired_cpus.zero?
-    desired_cpus = prov_resource.get_option(:number_of_sockets).to_i * prov_resource.get_option(:cores_per_socket).to_i
-  end
+  desired_cpus = requested_cpu(prov_resource, desired_nvms)
```

Comment:
@tinaafitz Then do we still need `:number_of_cpus` referenced in the `requested_cpu` method?  

Are there other fields that should be removed as well?  For example: `requested_memory` references `:memory` from get_options still which is not even a field name.  (Looks like it was incorrectly changed from `:vm_memory`). Is that an infra-only reference that should be removed as well?